source/features/multiple-accounts.js

```diff
@@ -0,0 +1,82 @@
+import {h} from 'dom-chef';
+import {getUsername, getUserImage} from '../libs/utils';
+
+const {sendMessage} = chrome.runtime;
+
+function getAccessTokens() {
+	return new Promise(resolve => {
+		sendMessage({message: 'reqAccessToken'}, response => {
+			resolve({token: response.token});
+			return true;
+		});
+	}).catch(err => console.error(err));
+}
+
+function createUserData() {
+	const userName = getUsername();
+	const userImage = getUserImage();
+
+	return getAccessTokens().then(data => ({
+		[userName]: {
+			image: userImage,
+			token: data.token
+		}
+	}));
+}
+
+function setLocalStorage() {
+	const {localStorage} = window;
```

Comment:
Don't use normal local storage. It's slow and limited. Use these instead: https://github.com/sindresorhus/refined-twitter/blob/ba7b04caab955f18ce67150a3be2c47ed06d64f2/source/libs/utils.js#L91-L105